indico/util/console.py

```diff
@@ -104,17 +103,18 @@ def _print_text(text):
 
 
 def _cformat_sub(m):
-    bg = 'on_{}'.format(m.group('bg')) if m.group('bg') else None
-    attrs = ['bold'] if m.group('fg_bold') else None
-    return colored('', m.group('fg'), bg, attrs=attrs)[:-4]
+    bold = '{b}' if m.group('fg_bold') else ''
+    bg = f"{{bg{m.group('bg')}}}" if m.group('bg') else ''
+    fg = f"{{{m.group('fg')}}}"
```

Comment:
I think this is one of the cases where f-strings do not improve readability (due to the need to escape the literal `{...}` in the string as `{{...}}`) - I'd consider using `%` formatting there, ie something like this:

```python
    bg = bold = ''
    if m.group('fg_bold'):
        bold = '{b}'
    if bg_color := m.group('bg'):
        bg = '{bg%s}' % bg_color
    fg = '{%s}' % m.group('fg')
    return Color(f'{bold}{bg}{fg}')
```

Reply:
Just when I thought I'd never use %-formatting again.. :D